compiler/src/dotty/tools/dotc/typer/RefChecks.scala

```diff
@@ -143,6 +143,7 @@ object RefChecks {
    *
    *    1.1. M must have the same or stronger access privileges as O.
    *    1.2. O must not be effectively final.
+   *    1.9.2  If M or O are extension methods, they must both be extension methods.
```

Comment:
I didn't think about if it makes sense in relation to the other checks.
Unless there are test to those numbers (`1.9.2`) have some order meaning. I did not saw any expressed logic for the order. 
Restrictions between the order of check should be documented.

Reply:
The ordering is just a notational device to align the coment and the code. We should change it to the new status. 

So:
```
   1.9.2 --> 1.3
   1.3 --> 1.4
   1.4 --> 1.5
   1.9.1 --> 1.9
```
We can keep it at that since 1.5 is currently unused.